ee/dtr/admin/configure/external-storage/index.md

```diff
@@ -4,52 +4,54 @@ description: Storage configuration for Docker Trusted Registry
 keywords: storage drivers, NFS, Azure, S3
 ---
 
+## Configure your storage backend
+
 By default DTR uses the local filesystem of the node where it is running to
 store your Docker images. You can configure DTR to use an external storage
 backend, for improved performance or high availability.
 
 ![architecture diagram](../../../images/configure-external-storage-1.svg)
 
-If your DTR deployment only has a single replica, you can continue using the
-local filesystem to store your Docker images. If your DTR deployment has
-multiple replicas, for high availability, you need to ensure all replicas are
-using the same storage backend. When a user pulls an image, the node serving
-the request needs to have access to that image.
+If your DTR deployment has a single replica, you can continue using the
+local filesystem for storing your Docker images. If your DTR deployment has
+multiple replicas, make sure all replicas are
+using the same storage backend for high availability. Whenever a user pulls an image, the DTR
+node serving the request needs to have access to that image.
 
-DTR supports these storage systems:
+DTR supports the following storage systems:
 
 * Local filesystem
-* NFS
-* Amazon S3 or compatible
-* Google Cloud Storage
-* Microsoft Azure Blob storage
-* OpenStack Swift
-
-To configure the storage backend, you can log into the **DTR web UI**
-as an administrator user, navigate to the **Settings** page, and choose
-**Storage**.
+   * NFS
+   * Bind Mount
```

Comment:
was not aware of this.  can one of the other reviewers fill me in with an example?

Reply:
Yes. If you configure storage with `--dtr-storage-volume <host_dir>` that directory will be bind mounted in instead of a volume. Similarly to how it works with `docker run -v`. It'll show up in the UI as

![screen shot 2019-03-07 at 1 50 22 pm](https://user-images.githubusercontent.com/706165/53991702-04ddaf80-40e0-11e9-8625-648a9929d76f.png)

Reply:
@trapier to get this PR out ASAP to go with the now published KB, let's take on refining this doc later on. I think the updated screenshot should be sufficient for now - then we can revisit this page later. I'll address all your review comments. Sound good?

Reply:
Yep.  Thanks!